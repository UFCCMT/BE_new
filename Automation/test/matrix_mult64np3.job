#!/bin/bash
#SBATCH -D /ufrc/lam/johnson2319/NGEE/Automation
#SBATCH --job-name=matrix_mult64np3.job
#SBATCH --mail-type=ALL
#SBATCH --mail-user=johnson@hcs.ufl.edu
#SBATCH --ntasks=3
#SBATCH --account=ccmt
#SBATCH --qos=ccmt
#SBATCH --mem-per-cpu=1gb
#SBATCH -t 00:5:00
#SBATCH -o test/matrix_mult64np3.out
#SBATCH -e test/matrix_mult64np3.err
 
module load intel/2016.0.109 openmpi/1.10.2
 
mpirun -np 3 ./mat_mult.o 64
