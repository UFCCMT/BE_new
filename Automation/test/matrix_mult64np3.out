./mat_mult.o
64
Sending 32 rows to task 1
Sending 32 rows to task 2
Sent all the data!
Receiving data from task 1
Receiving data from task 2
( 3 )time spent 0.000000 sec
